bokehjs/src/coffee/models/widgets/main.coffee

```diff
@@ -1,34 +1,36 @@
 module.exports = {
-  editors:                  [require('./cell_editors'), 'Editor']
-  formatters:               [require('./cell_formatters'), 'Formatter']
+  models: {
```

Comment:
Just to note, it's unlikely anyone will actually be affected by this change, because it would require him to `require()` widgets, which only happens once across the repository (in tests).